docs/release-notes/v0.32.0.md

```diff
@@ -0,0 +1,71 @@
+## Announcing Radius v0.32.0
+
+Today we're happy to announce the release of Radius v0.32.0. Check out the [highlights](#highlights) below, along with the [full changelog](#full-changelog) for more details.
+
+We would like to extend our thanks to all the [new](#new-contributors) and existing contributors who helped make this release possible!
+
+## Intro to Radius
+
+If you're new to Radius, check out our website, [radapp.io](https://radapp.io), for more information. Also visit our [getting started guide](https://docs.radapp.io/getting-started/) to learn how to install Radius and create your first app.
+
+## Highlights
+
+### WIP
```

Comment:
@willtsai @vinayada1 @lakshmimsft Do you have any features you want to highlight here ?

Reply:
adding @willtsai, @Reshrahim

Reply:
No feature added by me to highlight

Reply:
we can also highlight some of the community contributions - 

 1. Change open portal hyperlinks by @gpltaylor in https://github.com/radius-project/radius/pull/7229
2. chore: add dependabot configuration by @lechnerc77 in https://github.com/radius-project/radius/pull/7307